docs/UserGuide.md

```diff
@@ -1,41 +1,114 @@
+    * [About SweetRewards](#about-sweetrewards)
+    * [Getting Started with SweetRewards](#getting-started-with-sweetrewards)
+    * [Navigating this Guide](#navigating-this-guide)
+2. [Getting Started](#getting-started)
+3. [Features](#features)
+    * [Viewing help](#viewing-help--help)
+    * [Initializing program with seed data](#initializing-program-with-seed-data--seeddata)
+    * [Adding a Member](#adding-a-member--addmember)
+    * [Listing all members](#listing-all-members--list)
+    * [Adding Orders to a member](#adding-orders-to-a-member--addorder)
+    * [Adding points to a member](#adding-points--addpoints)
+    * [Updating Membership Tiers](#updating-membership--addmship)
+    * [Editing a member's details](#editing-a-member--edit)
+    * [Locating a member by name](#locating-members-by-name--find)
+    * [Deleting a member](#deleting-a-member--delete)
+    * [Clearing all entries](#clearing-all-entries--clear)
+    * [Exiting the program](#exiting-the-program--exit)
+4. [Glossary](#glossary)
+5. [FAQs](#faq)
+6. [Troubleshooting](#troubleshooting)
+7. [Command Summary](#command-summary)
+
+
+## Purpose of this Guide
+The purpose of this User Guide is to assist you in understanding and utilizing the SweetRewards application to its fullest potential. Whether you are looking to onboard new members, track orders, manage reward points or update membership tiers this guide will provide step-by-step instructions to ensure a smooth experience.
+
+## About SweetRewards
+SweetRewards is not just a contact management tool; it's your partner in cultivating customer loyalty and enhancing your bakery's membership program. With SweetRewards, you can:
+
+* Track member information: Keep a detailed record of each member, including their name, phone number, email, address, accumulated points, membership tier, and order history.
+* Manage orders and points: Seamlessly add orders and update points to reflect customer purchases and interactions.
+* Update membership tiers: Elevate members' tiers based on their accumulated points, unlocking new rewards and benefits aligned with each tier.
+
+### Understanding Membership Tiers
+One of the core features of SweetRewards is the Membership Tier System. This system rewards your loyal customers by categorizing them into different tiers based on the points they have accumulated through purchases at your bakery. Each tier offers different benefits, such as discounts, free items, or exclusive products. Here is a breakdown of the tiers:
+
+Bronze Tier (T1): This is the starting tier for all new members. Members need to accumulate at least 100 points to move to the next tier.
+Silver Tier (T2): Members who have accumulated 200 points are upgraded to the Silver Tier, which offers more benefits than the Bronze Tier.
+Gold Tier (T3): This is the highest tier, reserved for members who have collected 500 points. Gold Tier members receive the best benefits, underlining their importance to your bakery.
```

Comment:
I think for now can include Platinum tier first? 
```
    public static final String[][] TIERS = {
            {"PLATINUM", "1000"},
            {"GOLD", "500"},
            {"SILVER", "200"},
            {"BRONZE", "0"},
    };
```

If we change this at a later time then we'll update the UG accordingly ba

Reply:
Also I think we might not need the T1 T2 T3 stuff anym, like can just have BRONZE, SILVER etc. be the tier names

Reply:
ok nvm i get it this is for v1.2 LOL can js resolve first

Reply:
Yup! Im still trying to update it acc to v1.3 as much as I can for now to make life easier later so I'll implement these changes as well. Thanks for pointing out!